docs/troubleshooting.md

```diff
@@ -15,23 +15,16 @@ $ kubectl apply -f https://j.hept.io/contour-kuard-example
 
 ## Access the Envoy admin interface remotely
 
-By default the Envoy admin interface is bound to localhost.
-To access it remotely you can pass the flag `contour bootstrap --admin-address=0.0.0.0 /config/contour.yaml` in the `envoy-initconfig` init container for your Deployment or DaemonSet.
-This binds Envoy's admin interface to the Pod IP.
-(You can also pass `--admin-port=....` to change the port the admin interface is bound to).
+Getting access to the Envoy admin interface can be useful for diagnosing issues with routing or cluster health.
 
-If you wish to access the admin interface with kubeproxy, add the following stanza to your `contour` service object.
+The Envoy admin interface is bound by default to `http://127.0.0.1:9001`. 
+To access it from your workstation use `kubectl port-forward` like so,
 ```
-    - name: admin
-      port: 9001
-      protocol: TCP
+% kubectl -n heptio-contour port-forward contour-98d776469-xjw4p 9001
```

Comment:
Looks good to me.  

If you want we can have some bash magic here:
```bash
# Get one of the pods that matches the deployment/daemonset
CONTOUR_POD=$(kubectl -n heptio-contour get pod -l app=contour -o jsonpath='{.items[0].metadata.name}')
# Do the port forward to that pod
kubectl port-forward $CONTOUR_POD 9001
```

Reply:
even better for bash fu!